_posts/2021-05-04-how-to-update-your-ref-arch.adoc

```diff
@@ -112,9 +112,9 @@ context). As such, you need to inject these required blocks to use advanced feat
 root `terragrunt.hcl` configuration that includes these directives so that we can continue to use the original,
 non-Service Catalog based live configuration.
 
-For reference, you can download the link:assets/refarch-folder-structure.zip[refarch-folder-structure] zip file which
+For reference, you can download the https://github.com/gruntwork-io/terraform-aws-service-catalog/releases/tag/tbd[refarch-folder-structure TODO] zip file which
 contains all the following files in their respective locations in the folder structure. You'll have to copy these
-files into your own `infrastructure-live` repository.
+files into your own `infrastructure-live` repository. Note the new `_regional` folders under each region of each account; this folder was previously named `_global` and is the new home for region-level configurations, such as `sns-topics`.
```

Comment:
this will require a `terragrunt mv` won't it?

Reply:
The zip file (which includes those _regional dirs) was meant as an example for the whole Ref Arch, not just for the purposes of upgrading. For upgrading, the `_regional` can be ignored, because we want the easiest path forward. Now that we have the for-production examples updated, I don't think it's necessary for this zip file to serve that purpose, so probably best to remove the `_regional` dirs and also remove this text. Thanks @bendavies, prompting me to think through this.